database/mysql/mode/innodbbufferpoolhitrate.pm

```diff
@@ -103,12 +103,16 @@ sub run {
         $self->{output}->output_add(severity => $exit_code,
                                     short_msg => sprintf("innodb buffer pool hitrate at %.2f%%", $prcts{'bufferpool_hitrate' . ((defined($self->{option_results}->{lookback})) ? '' : '_now')})
                                     );
-        $self->{output}->perfdata_add(label => 'bufferpool_hitrate' . ((defined($self->{option_results}->{lookback})) ? '' : '_now'), unit => '%',
+        $self->{output}->perfdata_add(label => 'bufferpool_hitrate' . ((defined($self->{option_results}->{lookback})) ? '' : '_now'), 
+                                      nlabel => 'database.bufferpool_hitrate' . ((defined($self->{option_results}->{lookback})) ? '' : '_now') . '.percentage',
```

Comment:
Need to talk it the naming for that one (kind of particular with the now).

Reply:
As discussed, delta seems legit